build_containers/author-build.dockerfile

```diff
@@ -1,11 +1,12 @@
 FROM ubuntu:16.04
 
 RUN apt-get update \
-    && apt-get install -y curl wget git make build-essential python apt-transport-https xvfb libappindicator1 fonts-liberation openjdk-8-jre-headless
+    && apt-get install -y curl wget git make build-essential python apt-transport-https xvfb libappindicator1 fonts-liberation openjdk-8-jre-headless libgconf-2-4
 
 RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - \
     && echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list \
-    && curl -sL https://deb.nodesource.com/setup_7.x | bash -
+    && curl -sL https://deb.nodesource.com/setup_8.x | bash - \
+    && sh -c 'echo "deb https://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list'
 
 RUN apt-get update \
-    && apt-get install -y nodejs yarn
+    && apt-get install -y --allow-unauthenticated nodejs yarn google-chrome-stable
```

Comment:
Any reason not to add the [Google GPG key](https://www.google.com/linuxrepositories/) so we don't need to use `--allow-unauthenticated `?